`make docker_build_and_up` failed

An error occurred when starting up the Kubernetes scheduler simulator using Docker.

### environment

- M1 MacBook Pro
- Docker for Mac (darwin/arm64)
- docker engine 20.10.8

### Command

```
make docker_build_and_up
```

### Error Message

```
executor failed running [/bin/sh -c yarn build && yarn install --production --ignore-scripts --prefer-offline]: exit code: 1
make: *** [docker_build_front] Error 1
```

### Comment

/kind bug

--- a/README.md
+++ b/README.md
@@ -53,8 +53,8 @@ make docker_build_and_up
 
 Then, you can access the simulator with http://localhost:3000
 
-Note: 4GB of memory should be allocated to Docker.
-If you do not allocate enough memory, it may fail.
+Note: Insufficient memory allocation sometimes causes problems in building the image.
+Please allocate enough memory in that case.
 
 ### Run Locally
 

--- a/README.md
+++ b/README.md
@@ -53,6 +53,9 @@ make docker_build_and_up
 
 Then, you can access the simulator with http://localhost:3000
 
+Note: 4GB of memory should be allocated to Docker.
+If you do not allocate enough memory, it may fail.
+
 ### Run Locally
 
 You have to run frontend, server and etcd.

